### ISSUE TITLE
Dead links to release and its PDF on getpandas.html

### ISSUE THREAD

**Issue by grayed** [pandas-dev/pandas-website#86]
I've got here: https://pandas.pydata.org/getpandas.html
Then clicked "download" on the right, under "Release Candidate", and got 404. Same happens for "pdf" link. The "docs" link works fine.

I'm not a panda developer/user, just was crossing around, and was a bit surprised by invalid links on popular project's website. :)

**Comment by jorisvandenbossche** [comment]
It seems that none of the pdf links are working anymore, also not for older releases. cc @TomAugspurger

**Comment by TomAugspurger** [comment]
Yeah I’m not sure what’s up here. I thought I got it sorted out but apparently not.


I can take another look next week.

**Comment by TomAugspurger** [comment]
The other issue, about the "download" link is because I used 1.0.0 for the doc version, but we're in an RC so the version is actually 1.0.0rc0. So the link https://pypi.org/project/pandas/1.0.0/#files isn't valid.

**Comment by TomAugspurger** [comment]
For the PDFs, either nginx or cloudflare is messing up the redirects for PDFs specifically. I copied `pandas.pdf` to `foo.html` in the `0.25.3` docs, so 

```
$ wget -O pandas.pdf https://pandas.pydata.org/pandas-docs/version/0.25.3/foo.html
```

gets the PDF just fine.

**Comment by TomAugspurger** [comment]
Hmm, perhaps `/version/0.25.3/pandas.pdf` always worked fine? It's just when going through a symlink? The 1.0 version is a symlink from 1.0.0 -> 1.0.0rc0, so https://pandas.pydata.org/pandas-docs/version/1.0.0rc0/pandas.pdf is what actually works.

**Comment by TomAugspurger** [comment]
@grayed @jorisvandenbossche can you try the PDFs again? http://pandas.pydata.org/pandas-docs/stable/pandas.pdf is working for me now.

For the record, I added the lines

```
fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
fastcgi_param DOCUMENT_ROOT $realpath_root;
```

to fastcgi.conf and restarted nginx. Not sure if adding those lines actually did anything, or if it was just the restart...

**Comment by grayed** [comment]
> @grayed @jorisvandenbossche can you try the PDFs again? http://pandas.pydata.org/pandas-docs/stable/pandas.pdf is working for me now.

PDF downloads for stable releases are working now, yes.

The RC files link still gives 404, though, and RC PDF link causes `ERR_TOO_MANY_REDIRECTS`.

**Comment by jorisvandenbossche** [comment]
Also for the stable one: https://pandas.pydata.org/pandas-docs/stable/pandas.pdf, I still get "The page isn't redirecting properly"

**Comment by TomAugspurger** [comment]
Perhaps you’re hitting a cache somewhere... I’m not sure what’s going on.

**Comment by jorisvandenbossche** [comment]
Link is working now, so was probably due to the cache indeed

**Comment by TomAugspurger** [comment]
I'm seeing the redirect issues with https://pandas.pydata.org/pandas-docs/version/1.0.0/pandas.pdf again :/
